test/maybeRealPath-test.js

```diff
@@ -0,0 +1,48 @@
+var path = require('path');
+var proxyquire = require('proxyquire');
+var expect = require('unexpected')
+  .clone()
+  .addAssertion('<string> to resolve to <string>', function (expect, subject, value) {
+    return expect(subject, 'to equal', value);
```

Comment:
Huh, shouldn't this be `expect(maybeRealPath(fixturesPath(subject)), 'to equal', fixturesPath(value));` -- otherwise there's no real benefit to having a custom assertion?